db/suspensionReasons.ts

```diff
@@ -0,0 +1,14 @@
+import { SuspensionReason } from '../types/SuspensionReason';
+import knexConfig from '../knexfile';
+import knex from 'knex';
+
+const db = knex(knexConfig);
+
+const getAllForPsychologist = async (psychologistId: string) : Promise<SuspensionReason[]> => db
```

Comment:
If we want to follow the convention: `get` for all and `getOne` for one, I would suggest to call it `getByPsychologist`. Also it seems that `By` is more used than `For`. What do you think?

Reply:
GetBy suggest a select * group by
GetFor expect a param